rsrcdump/jsonio.py

```diff
@@ -3,18 +3,21 @@
 import base64
 import os
 import json
+from collections.abc import Mapping
 
 from rsrcdump.resconverters import ResourceConverter, Base16Converter
 from rsrcdump.textio import get_global_encoding, sanitize_type_name, sanitize_resource_name, parse_type_name
 from rsrcdump.resfork import Resource, ResourceFork
 
 
 class JSONEncoderBase16Fallback(json.JSONEncoder):
-    def default(self, o: Any):
+    __slots__ = ()
```

Comment:
I don't think `__slots__` are worth the maintenance overhead in a small project like this. Does mypy need them?

Reply:
No, but mypy will give you warnings when accessing attributes that don't exist in slots, and Python will use memory more efficiently when objects have `__slots__` defined